skype/skype-ps/skype/Set-CsTeamsAudioConferencingPolicy.md

```diff
@@ -0,0 +1,172 @@
+---
+external help file: MicrosoftTeams-help.xml
+Module Name: MicrosoftTeams
+online version: https://docs.microsoft.com/en-us/powershell/module/skype/set-csteamsaudioconferencingpolicy
+schema: 2.0.0
+---
+
+# Set-CsTeamsAudioConferencingPolicy
+
+## SYNOPSIS
+
+Audio conferencing policies can be used to manage audio conferencing toll- and toll-free numbers to be displayed in meeting invites created by users within your organization.
+
+## SYNTAX
+
+### Identity (Default)
+```
+Set-CsTeamsAudioConferencingPolicy [-AllowTollFreeDialin <Boolean>] [-Force] [[-Identity] <String>]
+ [-MeetingInvitePhoneNumbers <Object>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### Instance
+```
+Set-CsTeamsAudioConferencingPolicy [-AllowTollFreeDialin <Boolean>] [-Force]
+ [-MeetingInvitePhoneNumbers <Object>] [-Instance <PSObject>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The CsTeamsAudioConferencingPolicy cmdlet enables administrators to control audio conferencing toll- and toll-free numbers to be displayed in meeting invites created by users within your organization. The Set-CsTeamsAudioConferencingPolicy can be used to update an audio-conferencing policy that has been configured for use in your organization.
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+PS C:\> Set-CsTeamsAudioConferencingPolicy -Identity "EMEA Users" -AllowTollFreeDialin $False
+```
+
+In this example, AllowTollFreeDialin is set to false. All other policy properties will be left as previously assigned.
+
+### EXAMPLE 2
+```
+PS C:\> Set-CsTeamsAudioConferencingPolicy -Identity "EMEA Users" -AllowTollFreeDialin $True -MeetingInvitePhoneNumbers “+49695095XXXXX”,”+353156YYYYY”,”+1800856ZZZZZ”
+```
+
+In this example, two different property values are configured: AllowTollFreeDialIn is set to True and -MeetingInvitePhoneNumbers is set to include the following Toll and Toll free numbers - “+49695095XXXXX”,”+353156YYYYY”,”+1800856ZZZZZ” other policy properties will be left as previously assigned.
+
+## PARAMETERS
+
+### -AllowTollFreeDialin
+Determines if users of this Policy can have Toll free numbers. If toll-free numbers are available in your Microsoft Audio Conferencing bridge, this parameter controls if they can be used to join the meetings of a given user.
+
+```yaml
+Type: System.Boolean
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: True
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Force
+
```

Comment:
please include the description

Reply:
"Suppresses any confirmation prompts that would otherwise be displayed before making changes."